javaTask/src/Main.java

```diff
@@ -0,0 +1,59 @@
+import banking.*;
+
+import java.util.Scanner;
+
+public class Main {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Enter Account Holder Name : ");
+        String accountHolderName = scanner.next();
+        Account account = new Account();
+        account.setAccountHolderName(accountHolderName);
+        System.out.println("Enter Account User Name : ");
+        String accountUserName = scanner.next();
+        account.setAccountUserName(accountUserName + "@bank.com");
+        System.out.println("Enter Account password : ");
+        String password = scanner.next();
+        account.setPassword(password);
+        System.out.println("Account Holder Name : " + account.getAccountHolderName());
+        System.out.println("Account User Name : " + account.getAccountUserName());
+        System.out.println("Account Password : " + account.getPassword());
+
+
+        BankAccount bankAccount = new BankAccount();
+        System.out.println("set account number : ");
+        int accountNumber = scanner.nextInt();
+        bankAccount.setAccountNumber(accountNumber);
+        bankAccount.setAccountHolderName(accountHolderName);
+        bankAccount.setBalance(0);
+        System.out.println("Account number : " + bankAccount.getAccountNumber());
+        System.out.println("Account holder name : " + bankAccount.getAccountHolderName());
+        System.out.println("your current Balance = " + bankAccount.getBalance());
+        System.out.println("add money to your bank account : ");
+        int addMoney = scanner.nextInt();
+        bankAccount.depositingMoney(addMoney);
+        System.out.println("take money from your bank account : ");
+        int takeMoney = scanner.nextInt();
+        bankAccount.withdrawingMoney(takeMoney);
+
+        DepositTransaction depositTransaction = new DepositTransaction();
+        depositTransaction.performTransaction(addMoney);
+
+        WithdrawTransaction withdrawTransaction = new WithdrawTransaction();
+        withdrawTransaction.performTransaction(takeMoney);
+
+        SavingsAccount savingsAccount = new SavingsAccount();
```

Comment:
Same question as in line 24 and 51, as `SavingsAccount & CheckingAccount` are inheriting from Account.